src/frequenz/channels/_receiver.py

```diff
@@ -16,7 +145,7 @@
 
 
 class Receiver(ABC, Generic[_T]):
-    """A channel Receiver."""
+    """An entity that receives values from a channel."""
```

Comment:
I guess we need to suggest that we receive the values, but through these objects.  

Maybe something like this?

```suggestion
    """An type for receiving values from a channel."""

Reply:
Mmm, not sure, in general I think "A class for...", "A function for...", "An enum that..." all seems redundant when you already know the type of symbol you are dealing with, and this looks a bit about that. Asking for ideas to ChatGPT it suggests "endpoint", which seems better to me, so what about "An endpoint for receiving messages."?

Reply:
"Interface for receiving messages from a stream"?

Reply:
Yeah, that was another suggestion by ChatGPT, but I still find it too much like class/enum/etc, so I preferred endpoint. But if you prefer it, I could go with it.